spec/memo_wise_spec.rb

```diff
@@ -304,6 +293,17 @@ def self.with_positional_args(a, b) # rubocop:disable Naming/MethodParameterName
       end
     end
 
+    context "when instances are created with Class#allocate" do
+      let(:instance) { class_with_memo.allocate }
+
+      it "memoizes correctly" do
+        instance.no_args
+
+        expect(instance.no_args_counter).to eq(1)
+        expect(instance.no_args_counter).to eq(0)
+      end
+    end
```

Comment:
Did this test correctly fail without the change?

Reply:
This was a typo. You reviewed too soon! 😬